homeassistant/components/insteon/__init__.py

```diff
@@ -246,6 +265,12 @@ def load_aldb(service):
         else:
             _LOGGER.error('Entity %s is not an INSTEON device', entity_id)
 
+    def load_all_aldb(service):
+        """Load the ALDB for all devices."""
+        reload = service.data.get(SRV_LOAD_DB_RELOAD)
+        for entity in hass.data[DOMAIN].get('entities'):
+            entity.load_aldb(reload)
```

Comment:
Entities are a platform concern and we should avoid interacting with them directly outside of their platforms. I would recommend using our dispatch helper to send a signal to the entity to call the corresponding entity method.

See the tuya component for an example.

https://github.com/home-assistant/home-assistant/blob/1e149a704bb38866828920e186190d7d2e7d0586/homeassistant/components/tuya/__init__.py#L93-L95

https://github.com/home-assistant/home-assistant/blob/1e149a704bb38866828920e186190d7d2e7d0586/homeassistant/components/tuya/__init__.py#L118-L123

Reply:
Changed the `load_aldb` and `print_aldb` service too since they used the same approach.